mods/ra/rules/defaults.yaml

```diff
@@ -159,7 +159,13 @@
 
 ^AutoTargetGround:
 	AutoTarget:
+		AttackAnythingCondition: stance-attackanything
 	AutoTargetPriority@DEFAULT:
+		RequiresCondition: !stance-attackanything
+		ValidTargets: Infantry, Vehicle, Water, Defense
+		InvalidTargets: NoAutoTarget
+	AutoTargetPriority@ATTACKANYTHING:
+		RequiresCondition: stance-attackanything
 		ValidTargets: Infantry, Vehicle, Water, Structure, Defense
```

Comment:
Here as well.

Also, shouldn't there be an `AutoTargetWater` so that submerged but detected submarines can be targeted?